src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -40,7 +40,7 @@
         "symfony/dom-crawler": "^2.0.5",
         "symfony/intl": "~2.3",
         "symfony/security": "~2.6",
-        "symfony/form": "~2.7.25|^2.8.18",
+        "symfony/form": "~2.7.26|^2.8.19",
```

Comment:
needed to pull in https://github.com/symfony/symfony/pull/22066/files#diff-e11656489f1e01413320bd5edff0f0c7 to satisfy the `deps=low` job